core/src/test/java/io/narayana/nta/test/HandlerServiceTest.java

```diff
@@ -79,6 +79,7 @@ public static WebArchive createDeployment() {
                 .addAsWebInfResource(new FileAsset(new File("src/test/resources/persistence.xml")),
                         "classes/META-INF/persistence.xml")
                 .addAsManifestResource(new FileAsset(new File("src/test/resources/nta-test-ds.xml")), "nta-test-ds.xml")
+                .addAsManifestResource(new FileAsset(new File("src/test/resources/filter.properties")), "filter.properties")
```

Comment:
How about:
ClassLoader classLoader = getClass().getClassLoader();
File file = new File(classLoader.getResource("nta-test-ds.xml").getFile());

like you did in the getPropValues method above